src/shared/api/generated/CBioPortalAPIInternal.ts

```diff
@@ -270,6 +274,50 @@ export type GenesetMolecularData = {
 
         'value': string
 
+};
+export type Treatment = {
+    'treatmentId': string
+
+    'name': string
```

Comment:
sorry I'm actually not sure whether these changes are handwritten or not. if they are, you'll need to move them into another file as I mentioned above. if not, all good

Reply:
Oh really? I did not know. Since these files are in src/ I did not suspect this. I will rewrite all the code.

Reply:
Treatment API was replaced by swagger CodeGen output.